Kickstarter-iOS/Views/Controllers/PaymentMethodsViewController.swift

```diff
@@ -182,8 +182,18 @@ extension PaymentMethodsViewController: PaymentMethodsFooterViewDelegate {
 }
 
 extension PaymentMethodsViewController: AddNewCardViewControllerDelegate {
-  internal func presentAddCardSuccessfulBanner(_ message: String) {
-    self.viewModel.inputs.cardAddedSuccessfully(message)
+  func addNewCardViewControllerSucceeded(with message: String) {
+    self.dismiss(animated: true) { [weak self] in
+      guard let self = self else { return }
```

Comment:
`guard` might not be needed, we can just optionally chain here...either should be fine

Reply:
Hmm I would even consider testing this without the `weak` reference as I'm not sure this kind of closure would cause a retain cycle 🤔  but I guess it's good practice to just always do it.

Reply:
You're right! We're not holding a reference to this block so there should not be a retain cycle.